core/src/main/java/com/graphhopper/util/FetchWayGeometry.java

```diff
@@ -0,0 +1,20 @@
+package com.graphhopper.util;
+
+public enum FetchWayGeometry {
```

Comment:
I do not really like the enum name `FetchWayGeometry` too much. Its not a noun and it does not really tell us anything. Maybe we can use this name to communicate that this enum is used to specify how many nodes we fetch? For example `GeometryNodes.ALL`? `FetchMode.ALL`? `WayGeometryFetchMode.ALL`?

Reply:
Yes, FetchMode sounds already good.